### DISCUSSION TITLE
transferring wallet funds

### DISCUSSION THREAD

**Discussion by ghost** [btcpayserver/btcpayserver#5222] [General]
is it possible to transfer wallet funds with API without any payment pull or etc... ?

**Comment by pavlenex** [comment]
cc @ndeet @petzsch @Kukks

**Comment by mp21-org** [comment]
If you are using a Hot Wallet, you can create an onChain transaction with this [API call](https://docs.btcpayserver.org/API/Greenfield/v1/#operation/StoreOnChainWallets_CreateOnChainTransaction).
